### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by edfloreshz** [done-devs/done#96]
**Describe the feature you'd like**
When setting a reminder, the app should send notifications.

**Additional context**
Possible crate to use: [notify-rs](https://crates.io/crates/notify)

**Comment by gorentbarak** [comment]
This is a genuinely good idea.

**Comment by gorentbarak** [comment]
Can you assign me to this?

**Comment by edfloreshz** [comment]
Sure! Go nuts! 👍🏼

**Comment by gorentbarak** [comment]
Thanks.

**Comment by gorentbarak** [comment]
@edfloreshz Do you mean when setting a reminder, or when a reminder's date comes?

**Comment by edfloreshz** [comment]
The app needs to send a notification when a due date (including the time, if set) is the same as the user's clock.

**Comment by gorentbarak** [comment]
okay.

**Comment by gorentbarak** [comment]
@edfloreshz What are some notable files?

**Comment by edfloreshz** [comment]
There's a branch called `notifications`, you can start there.

**Comment by gorentbarak** [comment]
Okay.

**Comment by Johndjk** [comment]
The app doesn't show notifications?
And I had never come to see the github page thinking it was my ignorance in linux.
This feature needs to be implemented soon. It's essential!
The app is great, simple and to the point, however, without notifications it is very incomplete.

**Comment by edfloreshz** [comment]
I agree with you, this is one of the issues with the highest priority right now.
